### PR TITLE
Upgrade guide for release-7

### PR DESCRIPTION
**Pull request by rkothiya**

### CODE REVIEW

File: docs/release-notes/7.0.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
+# Release notes for Gluster 7.0
```
Comment (ShyamsundarR) [inline docs/release-notes/7.0.md]: I already see release notes for 7 live on the site https://docs.gluster.org/en/latest/release-notes/7.0/ is this a inadvertent repeat?

File: docs/release-notes/7.0.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+- Several stability fixes addressing,
+  - coverity, clang-scan, address sanitizer and valgrind reported issues
+  - removal of unused and hence, deprecated code and features
+- Performance Improvements
+
+
+Features 
+
+#### 1. Rpcbind not required in glusterd.service when gnfs isn't built.
```
Comment (ShyamsundarR) [inline docs/release-notes/7.0.md]: Reorder this.
(2) -> (1) Add text on what options are needed and what should be set to enable or use the feature
(3) -> (2) Is this by default, or are there options that enable this?
(1) -> (5) 
(4) remove issue # and if we have started an implementation, should it feature here? Are there ways in which users can test/use what is in the code base now? If so need documentation to help with the same

File: docs/release-notes/7.0.md (hunk 1 above)
Comment (rkothiya) [reply docs/release-notes/7.0.md]: I did not understand this. Isn't release notes suppose to be in gluster docs also ?
So it is there at two places : 
1. In gluster docs 
2. In glusterfs code base

File: docs/Upgrade-Guide/upgrade_to_7.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+# gluster volume reset <volname> <option>
+```
+
+**NOTE:** Failure to do the above may result in failure during online upgrades,
+and the reset of these options to their defaults needs to be done **prior** to
+upgrading the cluster.
+
+### Deprecated translators and upgrade procedure for volumes using these features
+
+With this release of Gluster, the following xlator/features are deprecated and
```
[the remark this answers is not in the document]
Comment (rkothiya) [reply docs/Upgrade-Guide/upgrade_to_7.md]: I have made the following changes let me know if it looks good now : 
[link to new draft of upgrade guide](https://hackmd.io/GTMWqysNQlKe_BbA66j1ng?view)

**Pull request merged**